### ISSUE TITLE
[Usage] 'Color' attribute is confusing

### ISSUE THREAD

**Issue by guillaumegarcia13** [glutengo/angular-mgl-timeline#5]
The 'color' attribute on **mgl-timeline-entry-dot** is actually confusing.
One may expect to fill a rgb or a hexadecimal value whereas the content is set as a class on the element.

If it actually is a class, why not simply use existing class or ngClass instead?

**Comment by glutengo** [comment]
I agree. I am only using it as a class because this way I can use the colors from the angular material theme (primary, accent). I‘ll consider changing it to ssomething like mat-color or simple ngClass support. 
Again, this can happen in two weeks time as I am currently on vacation. You can however make a proposal via a pull request in the meantime, if you want to. Thank you

**Comment by glutengo** [comment]
Attribute name was changed to color. Will be released on NPM in v 0.2.0 soon
